lib/ioki/apis/operator_api.rb

```diff
@@ -194,7 +194,7 @@ class OperatorApi
       Endpoints.crud_endpoints(
         :line_stop,
         base_path:   [API_BASE_PATH, 'products', :id, 'lines', :id],
-        except:      [:create, :delete],
+        except:      [:index, :show],
```

Comment:
This change is weird, why do we need it to support update?

Reply:
Yes, I believe we don't. But wasn't it completely wrong before? It's called `except`, but we included those actions we actually wanted.

Reply:
Makes sense, but it wasn't a separate commit, so I was wondering what the reason for this would be. Thanks for clarifying :)